.github/workflows/docs_latest.yml

```diff
@@ -18,6 +18,12 @@ jobs:
         python-version: ["3.12"]
         os: [ubuntu-latest]
     steps:
+      - name: Set routing name to latest
+        if: github.ref == 'refs/heads/main'
+        run: echo "REF_NAME=latest" >> "$GITHUB_ENV"
+      - name: Set routing name to test-XXXXXX
+        if: github.ref != 'refs/heads/main'
+        run: echo "REF_NAME=test-$(shuf -i 100000-999999 -n 1)" >> "$GITHUB_ENV"
```

Comment:
Explanation: If the workflow is triggered (manually, through workflow_dispatch) on another branch than the main-branch, then it will be published not under `/latest` but under `test-XXXXXX` where the X's are a 6-digit random number (starting with non-zero).

Reply:
magste das einfach als kommentar in die yml schreiben? wäre nachhaltiger

Reply:
https://github.com/bo4e/BO4E-python/pull/751/commits/e138f7affabf8b54c55f20bf05adeff4003c5c1e